index.html

```diff
@@ -2,9 +2,168 @@
 <html>
 <head>
 	<title>Sandwich Maker</title>
+	<link rel="stylesheet" type="text/css" href="Main.css">
 </head>
+
 <body>
+	<h2> Delicious Sandwich </h2>
+	<div id = "container" >
+
+		<form id= "breadForm" class="form">
+			<legend> Bread </legend>
+			<div id = "bread-chooser">
+				<label for="italian" > 
+					<input type="radio" name="bread" value="italian" id="italian"> Italian
+				</label>
+
+				<label for="flatbread" > 
+					<input type = radio name="bread" value="flatbread" id="flatbread"> Flat Bread
+				</label>
+
+				<label for="ryeBread" > 
+					<input type = radio name="bread" value="ryeBread" id="ryeBread"> Rye Bread
+				</label>
+
+				<label for="multiGrainFlatbread" > 
+					<input type = radio name="bread" value="multiGrainFlatbread" id="multiGrainFlatbread"> Multi Grain Flatbread
+				</label>
+
+				<label for="none" > 
+					<input type = radio name="bread" value="none" id="none"> No Bread
+				</label>
+				</div>
+		</form>
+
+		<form id= "meatForm" class="form" >
+			<legend> Meat </legend>	
+			<div id = "meat-chooser">
+				<label for="turky" > 
+					<input type = checkbox value="turky" id="turky"> Turky
+				</label>
+
+				<label for="chicken" > 
+					<input type = checkbox value="chicken" id="chicken"> Chicken
+				</label>
+
+				<label for="ham" > 
+					<input type = checkbox value="ham" id="ham"> Ham
+				</label>
+
+				<label for="bacon" > 
+					<input type = checkbox value="bacon" id="bacon"> Bacon
+				</label>
+
+				<label for="meatball" > 
+					<input type = checkbox value="meatball" id="meatball"> Meatball
+				</label>
+
+				<label for="none" > 
+					<input type = checkbox value="none" id="none"> No Meat
+				</label>
+			</div>
+		</form>
+
+		<form id= "cheeseForm" class="form">
+			<legend> Cheese </legend>
+			<div id = "cheese-chooser">
+				<label for="pepperJack" > 
+					<input type = checkbox value="pepperJack" id="pepperJack"> Pepper Jack
+				</label>
+
+				<label for="provolone" > 
+					<input type = checkbox value="provolone" id="provolone"> Provolone
+				</label>
+
+				<label for="shreddedMozzarella" > 
+					<input type = checkbox value="shreddedMozzarella" id="shreddedMozzarella"> Shredded Mozzarella
+				</label>
+
+				<label for="whiteAmerican" > 
+					<input type = checkbox value="whiteAmerican" id="whiteAmerican"> White American
+				</label>
+
+				<label for="montereyCheddar" > 
+					<input type = checkbox value="montereyCheddar" id="montereyCheddar"> Monterey Cheddar
+				</label>
+
+				<label for="none" > 
```

Comment:
You can only have 1 id element with the same name per page.  Currently you are using id="none" multiple times and it is causing some issues.  If you run the program and select a bread and then in the cheese section you click on the label for No Cheese it will select No Bread as well.